tests/test/application.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
         ["home-secondaryApplicant-maritalStatus", "Married"],
         ["home-secondaryApplicant-secondaryApplicantRelationship", "Spouse"],
         ["home-propertyType", "House"],
-        ["home-propertyStyle-23", "Yes"],
+        ["home-propertyStyle-2", "Yes"],
```

Comment:
This is the co-part to making all bowtie demos able to consume the existing integration test.